marlowe-cli/src/Language/Marlowe/CLI/Cardano/Api/PlutusScript.hs

```diff
@@ -27,6 +27,7 @@ import Language.Marlowe.CLI.Orphans ()
 withPlutusScriptVersion :: PlutusScriptVersion lang -> ((IsPlutusScriptLanguage lang) => a) -> a
 withPlutusScriptVersion PlutusScriptV1 = id
 withPlutusScriptVersion PlutusScriptV2 = id
+withPlutusScriptVersion PlutusScriptV3 = const $ error "unsupported until cardano-api exposes PlutusScriptV3"
```

Comment:
Should we also add a `FIXME` in situations like this? Just so we can easily `grep` to find this.

Reply:
We probably could - we could also grep for `PlutusScriptV3`